src/redux/sagas/utils/expendituresMulticall.ts

```diff
@@ -22,6 +22,8 @@ const EXPENDITURESLOTS_SLOT = toB32(BigNumber.from(26));
 const EXPENDITURESLOT_RECIPIENT = toB32(BigNumber.from(0));
 const EXPENDITURESLOT_CLAIMDELAY = toB32(BigNumber.from(1));
 
+const MAX_CLAIM_DELAY_VALUE = constants.MaxUint256;
```

Comment:
`constants.MaxUint128` or `constants.MaxUint256`? Previous value matched `constants.MaxUint128` if I remember correctly.

Reply:
It should be MaxUint256 according to https://github.com/JoinColony/colonyNetwork/blob/eb3620dbc1232e896f8e4d028889d058363cd0eb/contracts/colony/ColonyDataTypes.sol#L291